bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -462,4 +462,46 @@
 		<state pattern="%s"/>
 	</channel-type>
 
+	<channel-type id="sfp-present">
+		<item-type>Switch</item-type>
+		<label>SFP Present</label>
+		<category>Switch</category>
```

Comment:
To be checked but I believe we never set such category while the item type is Switch.
This will be the default value anyway.

Same for other added channel types.

Reply:
Would you prefer contact instead of switch ? They are read-only in any case.

Reply:
No, keep switch please.
I was wrong, this is sometimes provided like that, for example in the old Freebox binding.
So, let's keep this category.